keyboards/takashicompany/baumkuchen/keymaps/test/keymap.c

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 takashicompany (@takashicompany)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_Q, KC_W, KC_E, KC_R,
+        KC_A, KC_S, KC_D, KC_F,
+        KC_Z, KC_X, KC_C, KC_V, KC_B,
+                                KC_NO, KC_NO
+    )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    
+    if (record->event.pressed) {
+        rgblight_toggle();  // for test
```

Comment:
/cries in eeprom burn-through

```suggestion
        rgblight_toggle_neeeprom();  // for test
```